Functions.Templates/ProjectTemplate/CSharp-Isolated/Company.FunctionApp.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.3.0" OutputItemType="Analyzer"/>
-    <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.5.2" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.6.0" />
```

Comment:
Can you also make this change to f# project template and v3.x branch.

Reply:
I do not see the fsharp isolated template in this repo. 
Yes, Will have a PR targeting v3 as well.

Reply:
@soninaren PR targeting V3: https://github.com/Azure/azure-functions-templates/pull/1142